### DISCUSSION TITLE
Leaderboard Changes

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1065] [Ideas] [answered]
Adding on to discussion #1049, no other typists are able to get on the leaderboard except the gods of typing itself, so what if monkey type had a separate leaderboard for any typist that achieves extreme wpms(generally 200 or more). On this new leaderboard, players will need to type a test needing 100% accuracy as well. If a player fails to accomplish this, the test fails. This is almost exactly like setting your test difficulty to master, however you have others competing to get the highest wpm while still maintaining 100% accuracy. I have included an attachment to the text of which they could type. This text file needs improvements.

[impossibly hard typing excercise.txt](https://github.com/Miodec/monkeytype/files/6105264/impossibly.hard.typing.excercise.txt)

if you wish not to download the text file, here is the whole thing :
```
Learning the Keyboard
Type everything you see
This is a hard test you must take to get on the leaderboard
1. Type the home keys: asdf <space> jkl;
1. Type the home keys, capitalized, using the shift key: ASDF <space> JKL;
2. Move up and type the top row: qwertyuiop
3. Move down and type the bottom row: zxcvbnm,.
4. Type the alphabet, one letter at a time, trying not to look down.
Phrases to Practice
1. Pack my box with five dozen liquid diet cans or jugs.
2. Crazy Fredericka bought many very exquisite opal jewels.
3. Sixty zippers were quickly picked from the woven jute bag.
4. Amazingly few discotheques provide jukeboxes.
5. Heavy boxes perform quick waltzes and jigs.
6. Jackdaws love my big sphinx of quartz.
7. The five boxing wizards jump quickly.
8. How quickly daft jumping zebras vex.
9. Quick zephyrs blow, vexing daft Jim.
10. Sphinx of black quartz, judge my vow.
11. Waltz, nymph, for quick jigs vex Bud.
12. Blowzy night-frumps vex'd Jack Q.
13. Glum Schwartzkopf vex'd by NJ IQ.
14. The quick brown fox jumps over the lazy dog.
Other Exercises
1. this, they, the, thou, than, then, think, thank
2. as, at, all, am, an, app, ant, art, act, after
3. set, send, sat, sit, sip, sap, sop, sup
4. def, done, dan, din, dit, dot, dop
5. fit, fat, fop, fip, fup, fet, fey, fem, fun
6. gut, gun, gum, gux, guz, goz, get, git
7. hut, hit, hip, hap, him, hin, hib, hux, huz
8. jut, just, jup, jet, jen, jem, jex
9. kit, kat, kot, kut, kon, kom
10. list, last, lost, long, ling, lang 11. queen, quit, quince, quiet, quote
12. extra, exes, exit, exist, exotic
13. but, bun, bit, bip, bet, bert, bew
14. vet, vit, vot, vote, devout, devour 
python
15. print(“Hello, World!”)
java
16. System.out.println(“Hello, World!”);
javascript
17. console.log(“Hello, World!”);
```

**Comment by unrealapex** [comment, marked as answer]
seems a bit overboard

**Comment by Godje** [comment]
This is against what monkeytype essentially is.

Monkeytype is better than it's competitors because it doesn't take 2 minutes to complete a test. Master-mode leader-board would be cool. But taking a boring sequence test that would take over 3 minutes to do just once is simply against what is so great about monkeytype.
